rollup.config.js

```diff
@@ -1,34 +1,49 @@
-
-import buble from 'rollup-plugin-buble';
-import filesize from 'rollup-plugin-filesize';
-import resolve from 'rollup-plugin-node-resolve';
-import commonjs from 'rollup-plugin-commonjs';
-import uglify from 'rollup-plugin-uglify';
+import buble from "rollup-plugin-buble";
+import filesize from "rollup-plugin-filesize";
+import resolve from "rollup-plugin-node-resolve";
+import commonjs from "rollup-plugin-commonjs";
+import uglify from "rollup-plugin-uglify";
 import json from "rollup-plugin-json";
-import typescript from 'rollup-plugin-typescript';
+import typescript from "rollup-plugin-typescript";
+
+const MODULE_NAME = "AlgoliaAnalytics",
+  LIBRARY_OUTPUT_NAME = "search-insights";
 
-const MODULE_NAME = 'AlgoliaAnalytics', LIBRARY_OUTPUT_NAME = 'search-insights';
+const createPlugins = ({ browser }) => [
+  typescript(),
+  resolve({
+    browser,
+    preferBuiltins: false
+  }),
+  json({
+    preferConst: true,
+    compact: true
+  }),
+  buble(),
+  commonjs(),
+  uglify(),
+  filesize()
+];
 
-export default {
-  entry: 'lib/insights.ts',
-  format: 'umd',
-  moduleName: MODULE_NAME,
-  plugins: [
-    typescript(),
-    resolve({
-      browser: true,
-      preferBuiltins: false,
-    }),
-    json({
-      preferConst: true,
-      compact: true
-    }),
-    buble(),
-    commonjs(),
-    uglify(),
-    filesize()
-  ],
-  targets: [
-    { dest: `./dist/${LIBRARY_OUTPUT_NAME}.min.js`, format: 'umd' },
-  ],
-};
+export default [
+  {
+    input: "lib/insights.ts",
+    output: {
+      format: "umd",
+      name: MODULE_NAME,
+      file: `./dist/${LIBRARY_OUTPUT_NAME}.min.js`,
+      globals: {}
+    },
+    plugins: createPlugins({ browser: true })
+  },
+  {
+    input: "lib/insights.ts",
+    output: {
+      format: "cjs",
+      name: MODULE_NAME,
+      file: `./dist/${LIBRARY_OUTPUT_NAME}.cjs.min.js`
+    },
+    external: ["http"],
+    plugins: createPlugins({ browser: false })
```

Comment:
CommonJS technically could be used on the browser too, no?

Reply:
Hmm now that I see the doc again, I think I can just remove this option at all.

>      // some package.json files have a "browser" field which specifies
>      // alternative files to load for people bundling for the browser. If
>      // that's you, either use this option or add "browser" to the
>      // "mainfields" option, otherwise pkg.browser will be ignored

I've added `browser` field in `package.json`, so I think this isn't needed any more.
What do you think?

Reply:
it really depends on which dependencies we have, you need to check if they specify `browser` or are agnostic

Reply:
we have no dependencies, so `browser` doesn't need to be specified

Reply:
Here what do you mean by `dependencies`?  I don't understand what kind of dependencies can have something to do with `browser` field in package.json.